src/ussd_covid19_triage.js

```diff
@@ -399,20 +713,27 @@ go.app = (function () {
       var answers = self.im.user.answers;
 
       return new JsonApi(self.im).post(
-        self.im.config.eventstore.url + "/api/v2/covid19triage/", {
+        self.im.config.eventstore.url + "/api/v3/covid19triage/", {
         data: {
           msisdn: self.im.user.addr,
           source: "USSD",
           province: answers.state_province,
           city: answers.state_city,
+          city_location: answers.city_location,
           age: answers.state_age,
           fever: answers.state_fever,
           cough: answers.state_cough,
           sore_throat: answers.state_sore_throat,
           difficulty_breathing: answers.state_breathing,
+          smell: answers.state_taste,
```

Comment:
In the long run this will be confusing especially for someone touching the code base for the 1st time

Reply:
We can't change the name of the API field, but I can see if we can change the name of the state.